recipes/perfetto/all/test_package/CMakeLists.txt

```diff
@@ -4,5 +4,5 @@ project(test_package LANGUAGES CXX)
 find_package(perfetto REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```

Comment:
Should we condition this based on the version? Otherwise if you're stuck on an old compiler which doesn't support GCC 17 you wont be able to consume this package anymore. You should at least be able to consume the older versions

Reply:
Fixed.